frontend/src/scenes/data-pipelines/batch-exports/batchExportConfigurationLogic.tsx

```diff
@@ -51,6 +84,11 @@ export function getDefaultConfiguration(service: string): Record<string, any> {
             file_format: 'Parquet',
             compression: 'zstd',
         }),
+        ...(service === 'Redshift' && {
+            mode: 'COPY',
+            authorization_mode: 'IAMRole',
```

Comment:
What is the reasoning behind making `IAMRole` the default? I agree it's preferrable, but since the other authorization step doesn't support a role (it only accepts the usual AWS keys), it would require the user setting up both forms of authorization.

I'm just thinking as a user, I would likely set up a single 'batch exports' user, and grant it permissions to read/write from the specified bucket, then provide the key id and secret for this user. Not sure if adding another role makes anything more secure since we're already having to provide secret keys.

Reply:
> What is the reasoning behind making IAMRole the default?

It's the recommended one by AWS, in fact credentials are _very heavily_ discouraged.